timeline.md

```diff
@@ -9,26 +9,58 @@ css_id: timeline
 browser, Justin Samuel and Justin Cappos collaborate to design and publish an
 academic research paper on The Update Framework (TUF).
 
+**2011**: TUF Project moves to New York University Polytechnic
+(later NYU Tandon School of Engineering) when Justin Cappos accepts a post
+as an assistant professor in the Brooklyn, NY, school.
+
 **2013**: Justin Cappos, Trishank Kuppusamy, and Vladimir Diaz begin research
 into adapting and improving TUF for Python, Ruby, and other environments used
 for cloud computing.
 
+**2013**: The first of two Python Enhancement
+Proposals (PEP 458) dealing with TUF is published. "Surviving a Compromise of
+PyPI" details the integration of TUF into PyPI, the Python
+package manager.
+
 **2014**: Flynn.io independently implements TUF in the Go programming language.
```

Comment:
We should consider using just "Flynn."  Flynn.io is the URL for the project.

Flynn is also the first organization to implement TUF (other than the reference implementation).  I'm not sure if that's captured in the current sentence.

Reply:
A very good point...one I wasn't aware of or it would have certainly received more emphasis.